### ISSUE TITLE
Upgrading from 4.1.10 to 4.1.11 causes build command to break badly

### ISSUE THREAD

**Issue by brandonweiss** [middleman/middleman#2025]
This is weird, but when I upgrade from 4.1.10 to 4.1.11 and run `build` I get:

```
❯ be middleman build
== Executing: `brunch build`
20 Dec 21:43:49 - warn: source/stylesheets/app.scss compiled, but not written. Check your stylesheets.joinTo config
20 Dec 21:43:49 - warn: source/stylesheets/responsive.scss compiled, but not written. Check your stylesheets.joinTo config
20 Dec 21:43:49 - warn: source/stylesheets/github.css compiled, but not written. Check your stylesheets.joinTo config
20 Dec 21:43:49 - warn: source/stylesheets/utilities.scss compiled, but not written. Check your stylesheets.joinTo config
Autoprefixer's process() method is deprecated and will removed in next major release. Use postcss([autoprefixer]).process() instead
== External: 20 Dec 21:43:49 - info: compiled 2 files into 2 files in 1.3 sec
== Blog Sources: posts/{year}-{month}-{day}-{title} (:prefix + :sources)
Assertion failed: (!STR_EMBED_P(shared)), function str_new_frozen, file string.c, line 1075.
Abort trap: 6
```

I have no idea how to debug this… the [repo](https://github.com/brandonweiss/anti-pattern) it's happening on is public. Master is at 4.1.10.

**Comment by tdreyno** [comment]
Looks like an issue in Ruby itself. Are you on 2.3.0? This ticket implies it was fixed in 2.3.1: https://bugs.ruby-lang.org/issues/11946

**Comment by brandonweiss** [comment]
Ah, yep, that was it. Thanks!
